### PR TITLE
Performance test - GET

### PR DESCRIPTION
**Pull request by alex8937**
## Performance test
 To run the performance test locally:
 ```bazaar
 go run metric/performance.go
 ```
 To run the performance in the client container:
 ```bazaar
 make test-performance
 ```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -36,4 +36,7 @@
 .idea/
```
Comment (renga86) [inline .gitignore:36]: Hey Chen, thanks for doing this. 
can you pls update README with instructions as well on how to run this in our machines. (is it as simple as just running go perf_test.go ?)

Comment (alex8937) [reply .gitignore:36]: Updated the README.md

File: metric/performance.go [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+	"log"
+	"os"
+	"strconv"
+	"sync"
+	"time"
+)
+
+const (
+	localCoordAddr = "127.0.0.1:17000"
+	dockerCoordAddr = "node0:17000"
```
Comment (renga86) [inline metric/performance.go:17]: hmm, the "static mapping" have to be updated while running tests as well I guess for these cases. I have a static mapping of `dns/ip` names only. (if leader is down or different from `17000`, running from outside container mightn't work).

File: metric/performance.go [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+			clients = append(clients, client.NewRaftKVClient(coordAddr))
+		}
+		var wg sync.WaitGroup
+		for i, c := range clients {
+			wg.Add(1)
+			go func(c *client.RaftKVClient, k int) {
+				defer wg.Done()
+				key := strconv.Itoa(k)
+				expStart := time.Now()
+				for time.Since(expStart) < 5*time.Second {
```
Comment (renga86) [inline metric/performance.go:65]: minor nit: any time references can be made a constant so we can test various cases easily changing these numbers.

Comment (alex8937) [reply metric/performance.go]: Done.

File: metric/performance.go (hunk 2 above)
Comment (alex8937) [reply metric/performance.go:19]: Added a FIXME.

**Pull request merged**